Modelica/Thermal/HeatTransfer/Examples/TwoMasses.mo

```diff
@@ -23,7 +23,7 @@ equation
   connect(mass2.port, Tsensor2.port) annotation (Line(points={{70,20},{70,
           -60},{60,-60}}, color={191,0,0}));
 initial equation
-  T_final_K = (mass1.T*mass1.C + mass2.T*mass2.C)/(mass1.C + mass2.C);
+  T_final_K = Modelica.Constants.T_zero_K + (mass1.T*mass1.C + mass2.T*mass2.C)/(mass1.C + mass2.C);
```

Comment:
If we were to make it correct with offset the formula would be:
```suggestion
  T_final_K - Modelica.Constants.T_zero_K =  ((mass1.T-Modelica.Constants.T_zero_K)*mass1.C + (mass2.T-Modelica.Constants.T_zero_K)*mass2.C)/(mass1.C + mass2.C);
```
But the changes cancel out, so the original unchanged formula is simpler and cleaner.